udev/revpi_mac

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```
Should'n that be a POSIX script to run on all posix conform shells.

Reply:
Do you see any special use-case for this? Typically this script is only invoked by udev (see path) and that's it.

Reply:
And the Bash is compiled into udev? Of course on a Debian system, the bash is essential, but we have changed everything to CMake in our repositories under the aspect that everything should also be usable on other distributions.

Other scripts at the point also use /bin/sh.

Reply:
/bin/sh defaults to dash wich is the preferred shell for bootscripts on such on Debian

Reply:
> And the Bash is compiled into udev? Of course on a Debian system, the bash is essential, but we have changed everything to CMake in our repositories under the aspect that everything should also be usable on other distributions.

For portability of the script I would be in favor to use bash as the dash is more of a Debian thing and might not be present on other systems.

Reply:
Isn't POSIX invented exactly for this, @iluminat23? I mean 'For portability of the script...' 🤣

Reply:
> Isn't POSIX invented exactly for this, @iluminat23? I mean 'For portability of the script...' rofl

And then reality came by and dropped everything in the sink.

Reply:
> Isn't POSIX invented exactly for this, @iluminat23? I mean 'For portability of the script...' rofl

Since I use bash built-in calculation, I cannot declare it as POSIX script. The shebang is made deliberatly in a way to make it portable and don't implicitly use bash through a symlink.